spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/RabbitListenerEndpoint.java

```diff
@@ -116,6 +116,14 @@ default TaskExecutor getTaskExecutor() {
 	default void setBatchListener(boolean batchListener) {
 	}
 
+	/**
+	 * Whether this endpoint is for a batch listener.
+	 * @return {@link Boolean#TRUE} if batch.
+	 * @since 3.0
+	 */
+	@Nullable
+	Boolean getBatchListener();
```

Comment:
```suggestion
	Boolean isBatchListener();
```
?

Reply:
No; javaBean spec says `get` for `Boolean` `is` for `boolean`.